podcasts/AppDelegate+UrlHandling.swift

```diff
@@ -334,21 +334,17 @@ extension AppDelegate {
         }
 
         // Import OMPL extension
-        JLRoutes.global().addRoute("/import-opml/*") { [weak self] parameters -> Bool in
+        JLRoutes.global().addRoute("/import-file/*") { [weak self] parameters -> Bool in
             guard let self,
                   let rootViewController = SceneHelper.rootViewController(),
-                  let documentsURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first,
                   let originalUrl = parameters[JLRouteURLKey] as? URL else { return false }
 
-            let encodedOPML = originalUrl.absoluteString.replacingOccurrences(of: "pktc://import-opml/", with: "")
+            let opmlURLString = originalUrl.absoluteString.replacingOccurrences(of: "pktc://import-file/", with: "")
```

Comment:
Since this now can be an OPML, audio or movie, what do you think of:

```suggestion
            let fileURLString = originalUrl.absoluteString.replacingOccurrences(of: "pktc://import-file/", with: "")
```